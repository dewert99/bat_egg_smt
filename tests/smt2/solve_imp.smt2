(declare-const x Bool)
(declare-const y Bool)
(declare-const z Bool)
(declare-const a Bool)
(assert (=> x y z))
(check-sat-assuming (x a y (not z)))
(get-unsat-core)