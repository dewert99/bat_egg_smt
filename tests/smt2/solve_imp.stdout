Unsat
((not z) y x)
